### PR TITLE
[cmd] skip auto evaluation for none-detection dataset

### PR DESCRIPTION
**Pull request by IJtLJZ8Rm4Yr**
- [x] testing

### CODE REVIEW

File: ymir/command/mir/tools/mir_storage_ops.py [hunk 1]
```diff
@@ -51,13 +51,15 @@ def __build_task_keyword_context(cls, mir_datas: Dict['mirpb.MirStorage.V', Any]
...
         mir_tasks.tasks[mir_tasks.head_task_id].CopyFrom(task)
 
-        evaluation = det_eval_ops.det_evaluate_with_pb(
-            prediction=mir_annotations.prediction,
-            ground_truth=mir_annotations.ground_truth,
-            config=evaluate_config,
-            assets_metadata=mir_metadatas,
-        )
-        mir_tasks.tasks[mir_tasks.head_task_id].evaluation.CopyFrom(evaluation)
+        if mir_annotations.prediction.type == mir_annotations.ground_truth.type == mirpb.ObjectType.OT_DET_BOX:
```
Comment (fenrir-z) [inline ymir/command/mir/tools/mir_storage_ops.py:55]: 如果跳过，给个 warning

Comment (IJtLJZ8Rm4Yr) [reply ymir/command/mir/tools/mir_storage_ops.py:55]: cmd warning 怎么加？ 没看到 logger

**Review by phoenix-xhuang — DISMISSED** [review]

File: ymir/command/mir/tools/mir_storage_ops.py (hunk 1 above)
Comment (fenrir-z) [reply ymir/command/mir/tools/mir_storage_ops.py:55]: import logging
logging.warning

Comment (IJtLJZ8Rm4Yr) [reply ymir/command/mir/tools/mir_storage_ops.py:55]: added

**Review by phoenix-xhuang — APPROVED** [review]

**Review by fenrir-z — APPROVED** [review]

**Pull request merged**